portality/templates/layouts/dashboard_base.html

```diff
@@ -51,10 +51,22 @@ <h2 class="sr-only">DOAJ Dashboard</h2>
             <span data-feather="smile" aria-hidden="true"></span>
           </a>
         </li>
+        <li>
+          <a href="/" class="tag tag--secondary">
+            <span class="sr-only">Home</span>
+            <span data-feather="home" aria-hidden="true"></span>
+          </a>
+        </li>
+        <li>
```

Comment:
Is there any mileage in maybe including the entire front-end navigation system in a pull-down menu.  At least the parts that are built from configuration?  That way if I wanted to jump directly to the API docs, or the Journal search, I could go straight there rather than in 2 steps?

Reply:
We’ve already established this through the mockups — we want to diminish the noise that the current interface presents to users (all the public site menus etc are not relevant to work on the dashboard), so I want to go ahead with this and ensure that there is a clear distinction between public interface (searching journals and articles) vs dashboard interface (processing and managing applications); it’s central to the idea of the the editorial workflow overhaul. This may not be final, we’ll be testing navigation as soon as this is on the test server though.

Reply:
I'm probably a niche user, but it's very very common for me to be switching regularly between the admin, editorial, publisher and public views, especially when debugging.

Reply:
Ah that’s fair. Can we just remove it for now but I’ll carefully consider a design around that instead of just re-introducing the menu as it was? It’s possible that we just do like you suggested — collapse it in a dropdown — but I’d like to see exactly what parts of the menu would be useful. For example, having the journals and articles search sees repetitive. 

I’ll also see with the editorial team when we test.

Reply:
Yes, sure, no worries

Reply:
Resolving because I’ve noted this comment in testing documents